### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by griffobeid**
Without form validation or some other mechanism of controlling what we send to the backend the user could cause a bad subscription to nats which will continuously fail

image

Invalid:
image

Valid:
image

### CODE REVIEW

File: launch_chrome.sh [hunk 1]
```diff
@@ -6,11 +6,11 @@ SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
 
 # generate certificate
 
-openssl req -x509 -newkey rsa:2048 -keyout $SCRIPTPATH/actix-api/certs/localhost.key -out $SCRIPTPATH/actix-api/certs/localhost.pem -days 365 -nodes -subj "/CN=127.0.0.1"
+#openssl req -x509 -newkey rsa:2048 -keyout $SCRIPTPATH/actix-api/certs/localhost.key -out $SCRIPTPATH/actix-api/certs/localhost.pem -days 365 -nodes -subj "/CN=127.0.0.1"
```
Comment (darioalessandro) [inline launch_chrome.sh:9]: commented code

File: yew-ui/src/components/attendants.rs [hunk 2]
```diff
@@ -227,6 +227,7 @@ impl Component for AttendantsComponent {
                     info!("end to end encryption enabled = {}", self.e2ee_enabled);
                     let id = ctx.props().id.clone();
                     let email = ctx.props().email.clone();
+                    info!("email = {}", email);
```
Comment (darioalessandro) [inline yew-ui/src/components/attendants.rs:230]: do we need this log statements?

File: yew-ui/src/pages/home.rs [hunk 3]
```diff
@@ -29,11 +31,28 @@ pub fn home() -> Html {
...
-                    <input class="rounded-md mr-1 p-2 text-black" label="username" type="text" placeholder="Username" ref={username_ref}  />
-                    <input class="rounded-md ml-1 p-2 text-black" label="meeting_id" type="text" placeholder="Meeting ID" ref={meeting_id_ref} />
+                    <input
+                        class={TEXT_INPUT_CLASSES}
+                        label="username"
+                        type="text"
+                        placeholder="Username"
+                        ref={username_ref}
+                        required={true}
+                        pattern="^[a-zA-Z0-9_]*$"
```
Comment (darioalessandro) [inline yew-ui/src/pages/home.rs:45]: nice validation

**Review by darioalessandro — CHANGES_REQUESTED** [review]
@griffobeid looks good, I found some commented code and a log statement that looks like it was added for testing.

File: yew-ui/src/components/attendants.rs (hunk 2 above)
Comment (griffobeid) [reply yew-ui/src/components/attendants.rs]: Removed this

**Pull request merged**